profile/README.md

```diff
@@ -1,60 +1,89 @@
+- [Developer ergonomics](https://developers.flow.com/build/flow): Flow is designed to maximize developer productivity. Examples range from upgradeable smart contracts, native multi-sig support, and scripted transactions which allow a single transaction to interact with any number of smart contracts.
+- [Native on-chain randomness](https://flow.com/post/on-chain-randomness-on-flow): randomness is hard, and it’s even harder on-chain. Flow introduces a breakthrough ‘randomness beacon’ which generates an unbiased, unpredictable, and verifiable source of secure pseudo-randomness in a fully decentralized manner. This is essential for things like Gaming, DeFi and prediction markets without needing to rely on external Oracles. 
 
-### **Cadence**
+**Coming from other blockchains like Ethereum?** [**Check out our comparison guide**](https://developers.flow.com/build/building-vs-other-chain)
 
-[Cadence](https://github.com/onflow/cadence) is the first ergonomic, resource-oriented smart contract programming language. 
+## Get started on Flow
 
-While existing programming environments can be used to keep track of asset ownership, they are typically used in scenarios where they are _reflecting_ ownership rather than defining it directly. Public blockchains are unique in that they are explicitly designed to manage ownership of digital assets with scarcity and full access control. Digital assets on public blockchains behave like physical assets: they cannot be copied or counterfeited, only moved. 
- 
-‍[**Resource-oriented programming**](https://medium.com/dapperlabs/resource-oriented-programming-bee4d69c8f8e) is a new paradigm, designed to be secure and easy-to-use. For the first time, developers can create uniquely durable digital artifacts where ownership is tracked by the language itself, enabling a powerful new category of applications.
+Choose your path:
 
-As the first high-level resource-oriented programming language, Cadence has a comfortable, ergonomic syntax making it very easy to read. It uses a strong, static type system to minimize runtime errors, and allows all methods, interfaces, and transactions to include pre- and post-conditions to enforce expected behaviour. This has resulted in a language that is easier to learn, significantly easier to audit, and ultimately much more productive than any current alternatives. You can start learning Cadence on Flow Playground: [play.onflow.org](https://play.onflow.org/)
+**Build a Dapp on Flow**
```

Comment:
Add:

**Hello World on Flow**\
[Hello World] [I want to start with a Hello World on Flow.] (https://developers.flow.com/build/getting-started/quickstarts/hello-world)